cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -136,6 +150,21 @@ typedef uint32_t gpio_t;
 #define SPI_HWCS(x)         (SPI_HWCS_MASK | x)
 
 /**
+ * @brief   Map ADC device numbers to human readable strings
+ */
+enum {
+    ADC_1 = 0,          /**< ADC1 */
+#if defined(CPU_FAM_STM32F1) || defined(CPU_FAM_STM32F2)
+    ADC_2 = 1,          /**< ADC2 */
+#endif
+#if defined(CPU_MODEL_STM32F407VG) || defined(CPU_MODEL_STM32F415RG) \
+    || defined(CPU_MODEL_STM32F446RE) || defined(CPU_MODEL_STM32F429ZI)
+    ADC_2 = 1,          /**< ADC2 */
+    ADC_3 = 2           /**< ADC3 */
```

Comment:
ADC_3 is not defined for CPU_FAM_STM32F2 but is used in nucleo144-f207: the build fails. I tried to enable it for f207 but the application hangs when initializing ADC3...